packages/calling/src/CallSettings/CallSettings.ts

```diff
@@ -131,6 +144,7 @@ export class CallSettings implements ICallSettings {
 }
 
 /**
+ * Creates an instance of CallSettings Client.
```

Comment:
In fact, this method should be explained in detail and not the class definition above. This should also have the example code. Doesn't harm to keep it both the places but here, it seems more relevant.

Reply:
Will be done later

Reply:
Can we add a Todo with a spark ID in that case?

Reply:
@mkesavan13  Added comment in this Jira to take care of this change